LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Marcus Olsson
+Copyright (c) 2016 Marcus Olsson. Portions copyright (c) 2017 Google LLC.
```

Comment:
I don't see why this is necessary?

Reply:
I'm not a lawyer, but the guidance from my employer is to include this in OSS contribution unless there's specific clearance to assign copyright outside of Google. Their claim is that the employment contract assigns them copyright to even stuff I do on my own time.

I would _like to_ not include it as well- and I totally understand if you're not willing to accept contributions with this caveat. There is a process I can go through on my end to get the OK to skip this- it will just be some amount of time (at least time-not-on-the-weekend) for turnaround on my end.

Note that, in my understanding, this doesn't change the licensing; it's still open-source, MIT-licensed, etc.

Reply:
Hm, I don't have enough knowledge in these matters, but right now I don't feel good about including it. Please allow me to do some reading before I decide on this.

Reply:
It may be more advantageous to build a CLA for this contributor to sign, or wait for them to go through the process for asking permission from their employer to have a personal life.

Reply:
Sure thing- I understand your reluctance. :-) I've sent a request in on my end to get an explicit release.

(Also, @theckman - you're the fourth /.*e[c]?kman/ I know of in SRE. What's up with that?)

Reply:
@cceckman If you don't mind then, let's wait for the request to come through. I'll consider adopting a CLA, but in the meantime, I hope that will be sufficient. I really appreciate you going through the trouble! Looking forward working with you on this! And thanks @theckman for your input!

Reply:
Just to close out this thread- I do have an explicit copyright release from the company now, so expect to see future PRs without this annotation. :-)